### PR TITLE
Note indicating Node location not a DTR upgrade factor.

### PR DESCRIPTION
**Pull request by KoryKessel-Docker**
In response to Issue #7319.

### CODE REVIEW

File: ee/dtr/admin/upgrade.md [hunk 1]
```diff
@@ -34,72 +27,74 @@ guarantees while upgrading between versions. While downgrades are not supported,
...
-Make sure you have at least [16GB of available RAM](install/system-requirements) on the node you are running the upgrade on. If the DTR node does not have access to the internet, you can
-follow the [offline installation documentation](install/install-offline)
-to get the images.
+Confirm that at least [16GB RAM is available](install/system-requirements) on the node on which you are running the upgrade. If the DTR node does not have access to the internet, follow the [offline installation documentation](install/install-offline) to get the images.
 
 Once you have the latest image on your machine (and the images on the target
-nodes if upgrading offline), run the upgrade command:
+nodes, if upgrading offline), run the upgrade command.
+
+> On which node should the upgrade command be run?
```
Comment (traci-morrison) [inline ee/dtr/admin/upgrade.md]: Add the Note label.

File: ee/dtr/admin/upgrade.md [hunk 2]
```diff
@@ -34,72 +27,74 @@ guarantees while upgrading between versions. While downgrades are not supported,
...
+>
+> The upgrade command can be run from any available node, as UCP is aware of of which worker nodes have replicas.
 
 ```bash
 docker run -it --rm \
   {{ page.dtr_org }}/{{ page.dtr_repo }}:{{ page.dtr_version }} upgrade
 ```
 
-By default the upgrade command runs in interactive mode and prompts you for
+By default the upgrade command runs in interactive mode and prompts for
```
Comment (traci-morrison) [inline ee/dtr/admin/upgrade.md]: "By default, ..."

File: ee/dtr/admin/upgrade.md (hunk 1 above)
Comment (KoryKessel-Docker) [reply ee/dtr/admin/upgrade.md]: Done. Thanks.

File: ee/dtr/admin/upgrade.md [hunk 3]
```diff
@@ -34,72 +27,74 @@ guarantees while upgrading between versions. While downgrades are not supported,
...
+When upgrading from `2.5` to `2.6`, the system will run a `metadatastoremigration` job following a successful upgrade. This involves migrating the blob links for your images, which is necessary for online garbage collection. With `2.6`, you can log into the DTR web interface and navigate to **System > Job Logs** to check the status of the `metadatastoremigration` job. Refer to [Audit Jobs via the Web Interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/) for more details.
 
 ![](../images/migration-warning.png){: .with-border}
 
 Garbage collection is disabled while the migration is running. In the case of a failed `metadatastoremigration`, the system will retry twice.
 
 ![](../images/migration-error.png){: .with-border}
 
-If the three attempts fail, you will have to retrigger the `metadatastoremigration` job manually. To do so, send a `POST` request to the `/api/v0/jobs` endpoint:
+If the three attempts fail it will be necessary to manually retrigger the `metadatastoremigration` job. To do this, send a `POST` request to the `/api/v0/jobs` endpoint:
```
Comment (traci-morrison) [inline ee/dtr/admin/upgrade.md]: "fail, it will be..."

File: ee/dtr/admin/upgrade.md (hunk 2 above)
Comment (KoryKessel-Docker) [reply ee/dtr/admin/upgrade.md]: Done. Thanks.

File: ee/dtr/admin/upgrade.md (hunk 3 above)
Comment (KoryKessel-Docker) [reply ee/dtr/admin/upgrade.md]: I don't really agree, but as adding the comma doesn't make it wrong...letcha have it. :-)

**Pull request merged**